src/presentation/pages/Neoga/Result/index.tsx

```diff
@@ -25,8 +26,18 @@ function NeogaResult() {
       </Link>
       <h1>{username}님이 만든 너가소개서</h1>
       <h2>내가 생성한 너가소개서의 답변을 확인하세요</h2>
-      {cardItem &&
-        cardItem.resultList.map((result) => <NeogaResultCard key={result.id} {...result} />)}
+      {replyList && replyList.map((card) => <NeogaResultCard key={card.id} {...card} />)}
+      {noReplyList &&
+        noReplyList.map(({ id, title, subtitle, darkIconImage }) => (
+          <NeogaCreateCardItem
+            key={id}
+            idx={id}
+            title={title}
+            content={subtitle}
+            src={darkIconImage}
+            onClick={() => navigate(`/neoga/${id}/detail/form`)}
+          />
+        ))}
```

Comment:
위에서 filter하는 것 대신에 여기서 삼항 연산자를 쓰는 것도 좋을 것 같아요!

```suggestion
      {cardItem.resultList.map((card) => 
      	 card.answer.length > 0 
      	   ? <NeogaResultCard key={card.id} {...card} />)}
      	   : <NeogaCreateCardItem
	              key={id}
	              idx={id}
	              title={title}
	              content={subtitle}
	              src={darkIconImage}
	              onClick={() => navigate(`/neoga/${id}/detail/form`)}
	            />
        )}
```

Reply:
우왕 꼼꼼한 리뷰 감사합니다 저두 이 방법이 더 좋아요 !!! 💟